README.md

```diff
@@ -1,3 +1,5 @@
-# ChelFs
+# Chel-FS
 
-A distirbuted filesystem on DAOS object Store.
+## Introduction
+
+Chel-FS is a high performance filesystem built on top of DAOS, a mordern flash medium first storage pool. Chel-FS is aimed for workloads that require high performance filesystem operations. Chel-FS is a layer that is built on top of DAOS. Only filesystem metadata operations will go through Chel-FS, user data will directly flow between DAOS servers and clients. In such a way we can have both high IO bandwidth and IOPS.
```

Comment:
It's a High Performance Distributed Filesystem

Reply:
Scaling is our important feature.

Reply:
Accepted